src/digraph.rs

```diff
@@ -984,6 +984,11 @@ impl PyDiGraph {
     pub fn add_edge(&mut self, parent: usize, child: usize, edge: PyObject) -> PyResult<usize> {
         let p_index = NodeIndex::new(parent);
         let c_index = NodeIndex::new(child);
+        if !self.graph.contains_node(p_index) || !self.graph.contains_node(c_index) {
```

Comment:
I'm wondering if we should move this check to `_add_edge()` so that it covers all the edge addition methods. Like if we did:

```
graph = rx.PyDiGraph()
graph.add_edges_from([(0, 1, None), (0, 2, None)])
```
we'd get the same panic.

Reply:
Done